packages/components/tabs/src/tab.tsx

```diff
@@ -78,8 +78,8 @@ const Tab = forwardRef<"button", TabItemProps>((props, ref) => {
     rerender: true,
   });
 
-  const handleClick = () => {
-    chain(onClick, tabProps.onClick);
+  const handleClick = (...args: any[]) => {
+    chain(onClick, tabProps.onClick)(...args);
```

Comment:
can you share some examples of args?

Reply:
No, just ensure that the DOM event can be smoothly passed through.

Reply:
Like this

<img width="803" alt="image" src="https://github.com/user-attachments/assets/7e3b1d05-5361-4718-8a73-1329e905e2b6">

Reply:
This's for `tabProps.onClick`. There should be no args for `handleClick`.

Reply:
Here, `handleClick` did not receive `event`, so `tabProps.onClick` will not have `event` either.

Reply:
please add a test case to cover this change as well.

Reply:
already updated
```
await user.click(tab1);
expect(item1Click).toHaveBeenCalled();
expect(item1Click.mock.lastCall[0].target).toBe(tab1);

await user.click(tab2);
expect(item2Click).toHaveBeenCalled();
expect(item2Click.mock.lastCall[0].target).toBe(tab2);
```

Reply:
I removed the event parameter and updated the tests; please take a look when you have time.